mk/check.mk

```diff
@@ -45,7 +45,7 @@ format: fmt generate docs tidy ginkgo/unfocus ginkgo/lint
 
 .PHONY: kube-lint
 kube-lint:
-	$(KUBE_LINTER) lint .
+	$(KUBE_LINTER) lint deployments/charts/kuma
```

Comment:
Should you also check `app/kumactl/cmd/install/testdata/**/*.golden.yaml` so that all outputs of the helm charts are valid yamls too?

Reply:
I think kube-lint has some understanding of helm charts, that the directory being checked is a helm chart. I can double check though

Reply:
Looking at https://github.com/stackrox/kube-linter/issues/48 it looks like we should check the output

Reply:
OK, I'll do this in a separate PR :grin:

Reply:
Ok it was my feeling too :)